caim_base/static/js/common.js

```diff
@@ -46,6 +46,47 @@ $('[data-timestamp]').each(function (idx, el) {
     el.text(fmt.format(dte));
 });
 
+$('.reply').click(function(button){
+    let buttonID = button.target.id;
+
+        // Reply form element
+    var replyForm = `
+    <form method="post" id="reply-form${buttonID}" class="reply-form" onsubmit="submitForm(event)">
+        <input type="text" name="comment_id" value="${buttonID}" hidden>
```

Comment:
Carried over from previous PR, this should be `<input type="hidden">` not type="text" hidden

Reply:
Great point, will fix this.